discovery/src/main/java/org/openedx/discovery/presentation/WebViewDiscoveryViewModel.kt

```diff
@@ -0,0 +1,19 @@
+package org.openedx.discovery.presentation
+
+import org.openedx.core.BaseViewModel
+import org.openedx.core.config.Config
+
+class WebViewDiscoveryViewModel(
+    config: Config,
+) : BaseViewModel() {
+
+    private var _discoveryUrl = config.getDiscoveryConfig().webViewConfig.baseUrl
```

Comment:
Please do not use var with composables, please try to use MutableStateFlow<String> here

Reply:
It's not a composable and needs to be updated on every webView session. 🤔

Reply:
i mean that it is in use here:
```
WebViewDiscoverScreen(
             windowSize = windowSize,
             url = viewModel.discoveryUrl,
             cookieManager = edxCookieManager,
```
so here is an example of what I mean, here:
```
    private val _discoveryUrl = MutableStateFlow(config.getDiscoveryConfig().webViewConfig.baseUrl)
    val discoveryUrl = _discoveryUrl.asStateFlow()
```
and in WebviewDiscoveryFragment:
```
val discoveryUrl by viewModel.discoveryUrl.collectAsState()
Surface {
    Box(
        modifier = Modifier
            ....
    ) {
        if (hasInternetConnection) {
            WebViewDiscoverScreen(
            ....
                url = discoveryUrl,
```

Reply:
Discussed over Huddle. No need to update for now.